### ISSUE TITLE
Unity SDK not working properly on windows.

### ISSUE THREAD

**Issue by USAMAWIZARD** [ant-media/Ant-Media-Server#5480]
### Short description
Unity SDK not working properly on windows.

### Environment

* Operating system and version:
* Java version: 
* Ant Media Server version:
* Browser name and version:

### Steps to reproduce
1. run the publish example 
2. set the websocket url 
3. hit publish button 
4.stream will be in broadcasting but not playable 

Adaptive streaming was enabled with VP8 and h264

### Logs
__Place logs on [pastebin](http://pastebin.com/) or elsewhere and put links here__

Ask your questions on [Ant Media Github Discussions](https://github.com/orgs/ant-media/discussions)

**Comment by burak-58** [comment]
@USAMAWIZARD can you try it when the Unity Editor Window is at the top. It works if it is at top.

**Comment by USAMAWIZARD** [comment]
@burak-58  abi I tried with Unity window on top and it still did not work for me 

This is how I tested let me know if I did something wrong [here  ](https://www.veed.io/embed/2012e87e-4dd0-42f4-ab99-2adbf42ccbf0)

**Comment by burak-58** [comment]
@USAMAWIZARD have you ever tried with only VP8?

**Comment by USAMAWIZARD** [comment]
Yes I tried with only Vp8 I had disabled the h264 in app settings
